docs/analysis/dependency-map.md

```
@@ -0,0 +1,1432 @@
+        CodebaseChat[Codebase Chat]
+        InlineEdit[Inline Edit]
+        WorkspaceRAG[Workspace RAG]
+        MCPExt[MCP Extension]
+    end
+
+    subgraph "Shared Libraries"
+        CLI[VibeCode CLI<br/>packages/vibecode-cli]
+        Types[TypeScript Types<br/>types/]
+    end
+
+    %% Frontend dependencies
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)